src/sentry/static/sentry/less/misc.less

```diff
@@ -16,6 +16,24 @@ img {
 	white-space: nowrap;
 }
 
+.break-word {
+  /* These are technically the same, but we have to use both */
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+
+  -ms-word-break: break-all;
+  /* This is the dangerous one in WebKit, as it breaks things wherever */
+  word-break: break-all;
+  /* Instead use this non-standard one: */
+  word-break: break-word;
+
+  /* Adds a hyphen where the word breaks, if supported */
```

Comment:
Will the copy/paste include hyphens? I'm just saying, as a developer, I'd rather see the original message/exception and not be confused by an additional hyphen that was introduced  for aesthetic purposes.